scbasset/model.yaml

```diff
@@ -0,0 +1,59 @@
+type: keras
+args:
+  arch: model.json
+  weights:
+    md5: 2f6d44c9ccb579295700859aaee697bc
+    url: https://storage.googleapis.com/scbasset_tutorial_data/buen_model.h5
+  custom_objects: custom_objects.py
+default_dataloader:
+  defined_as: kipoiseq.dataloaders.SeqIntervalDl
+  default_args:
+    alphabet_axis: 1
+    auto_resize_len: 1344
+    dtype: np.float32
+dependencies:
+  conda:
+  - python=3.7
+  - tensorflow==2.6.0
```

Comment:
It should be `tensorflow=2.6.0`. Since the tests are passing anyway, I 'd say never mind. Just for future reference :)